tests/ashley/api/test_api_manage_moderators.py

```diff
@@ -211,13 +212,184 @@ def test_access_can_manage_moderators_list_non_moderators(self):
         # Permission + session added, it should be allowed
         self.assertEqual(response.status_code, 200)
 
-    def test_access_api_can_manage_moderators_update_student_promote(self):
+    def test_access_api_update_student_patch(self):
+        """Standard call should not be allowed to update a student."""
+        # Creates a forum
+        user = UserFactory()
+        lti_context = LTIContextFactory(lti_consumer=user.lti_consumer)
+        forum = ForumFactory()
+        forum.lti_contexts.add(lti_context)
+        assign_perm("can_read_forum", user, forum, True)
+
+        # Creates the session
+        self.client.force_login(user, "ashley.auth.backend.LTIBackend")
+        session = self.client.session
+        session[SESSION_LTI_CONTEXT_ID] = lti_context.id
+        session.save()
+        data = {
+            "public_username": "toto",
+        }
+        response = self.client.patch(
+            f"/api/v1.0/users/{user.id}/",
+            json.dumps(data),
+            content_type="application/json",
+        )
+        self.assertEqual(response.status_code, 403)
+        self.assertEqual(
+            response.json(),
+            {"detail": "You do not have permission to perform this action."},
+        )
+
+    def test_access_api_update_instructor_post(self):
+        """Post to update user is not allowed even for an instructor."""
+        # Creates a forum
+        user = UserFactory()
+        lti_context = LTIContextFactory(lti_consumer=user.lti_consumer)
+        forum = ForumFactory()
+        forum.lti_contexts.add(lti_context)
+        lti_context.sync_user_groups(user, ["instructor"])
+        assign_perm("can_manage_moderator", user, forum, True)
+        # Creates the session
+        self.client.force_login(user, "ashley.auth.backend.LTIBackend")
+        session = self.client.session
+        session[SESSION_LTI_CONTEXT_ID] = lti_context.id
+        session.save()
+        response = self.client.post(
```

Comment:
Note that POST should be on the list endpoint. That's why you get a 405. 
But ok you may want to confirm that this endpoint does not accept POST requests.